proto/bpfman.proto

```diff
@@ -140,22 +147,23 @@ message UprobeAttachInfo {
     bool retprobe = 4;
```

Comment:
Do we still need retprobe in the attach info if it's also in the ProbeLoadInfo?

Reply:
Fixed